### PR TITLE
Add events on Ingress detail view

### LINKED ISSUE #6407: Events on Ingress View
##### What would you like to be added
List Ingress Events on the View Ingress page

##### Why is this needed
If Events are created in Kubernetes when creating or maintaining an Ingress object, they cannot be viewed in the dashboard.

##### Comments
Self-Explanatory.  Add Events data table like on other pages in the dashboard.

### PR DESCRIPTION
**Pull request by klenje**
Fixes #6407 

image

### CODE REVIEW

File: src/app/backend/handler/apihandler.go [hunk 1]
```diff
@@ -1053,6 +1057,25 @@ func (apiHandler *APIHandler) handleGetIngressDetail(request *restful.Request, r
...
+	k8sClient, err := apiHandler.cManager.Client(request)
+	if err != nil {
+		errors.HandleInternalError(response, err)
+		return
+	}
+
+	namespace := request.PathParameter("namespace")
+	name := request.PathParameter("ingress")
+	dataSelect := parser.ParseDataSelectPathParameter(request)
+	dataSelect.MetricQuery = dataselect.StandardMetrics
```
Comment (floreks) [inline src/app/backend/handler/apihandler.go:1070]: I'd use here `dataselect.NoMetrics` as events do not have metrics associated with them AFAIK.

Comment (klenje) [reply src/app/backend/handler/apihandler.go]: make sense, thanks. Changed that

**Comment by klenje** [comment]
looks like E2E is broken, it was ok in my local tests, are you/someone working on fixing them? I guess this blocks the merge even if not linked to this specific PR @floreks

**Comment by floreks** [comment]
I think e2e tests were fixed. Could you rebase?

**Comment by klenje** [comment]
> I think e2e tests were fixed. Could you rebase/merge master?

@floreks rebased, let's see if tests have better results now

**Comment by klenje** [comment]
rebasing helped to have green tests @floreks :) let me know if anything else is missing

**Pull request merged**